src/main/java/org/apache/commons/collections4/CollectionUtils.java

```diff
@@ -1266,9 +1248,6 @@ public static Object get(final Object object, final int index) {
         } else if (object instanceof Iterable<?>) {
             final Iterable<?> iterable = (Iterable<?>) object;
             return IterableUtils.get(iterable, i);
-        } else if (object instanceof Collection<?>) {
```

Comment:
Nice. Will check out the code to review with the IDE, plus some coffee :zzz: I think it looks good to be merged. Thanks for the quick replies @izebit